bedrock/firefox/views.py

```diff
@@ -560,7 +560,6 @@ class WhatsnewView(L10nTemplateView):
         'firefox/whatsnew/whatsnew-fx80.html': ['firefox/whatsnew/whatsnew-fx80', 'firefox/whatsnew/whatsnew'],
         'firefox/whatsnew/whatsnew-fx81.html': ['firefox/whatsnew/whatsnew-fx81', 'firefox/whatsnew/whatsnew'],
         'firefox/whatsnew/whatsnew-fx82.html': ['firefox/whatsnew/whatsnew-fx80', 'firefox/whatsnew/whatsnew'],
-        'firefox/whatsnew/whatsnew-fx83.html': ['firefox/whatsnew/whatsnew-fx83', 'firefox/whatsnew/whatsnew'],
```

Comment:
Doh, looks like we do need `firefox/whatsnew/whatsnew` for the up-to-date notification, and will need it for both the English and German templates.

![image](https://user-images.githubusercontent.com/205591/98136248-95503280-1e75-11eb-8ba9-9cbd986953ab.png)

Reply:
Ahhhh! Great catch.

Reply:
Hmm... won't this mess with locale activation? (first file in the array is used to activate locales). Perhaps worth testing using `Dev=False` in your `.env`.

Reply:
I added back the `firefox/whatsnew/whatsnew` FTL file for the template.

Also, I need to set them for both `whatsnew-fx83-en.html` and `whatsnew-fx83-de.html`.

Reply:
Working for me in local prod mode:

![image](https://user-images.githubusercontent.com/205591/98138069-92564180-1e77-11eb-98f9-f0339c2084c9.png)

Reply:
Right, the copy shows ok - but the page says it's available in many more languages (dictated by `firefox/whatsnew/whatsnew.ftl`):

<img width="387" alt="image" src="https://user-images.githubusercontent.com/400117/98138608-31505d00-1ebb-11eb-8d62-517a354238a3.png">

Reply:
As @craigcook pointed out in Slack, looks like this is OK due to the `en` conditional in the view :)